ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -38,7 +38,7 @@ tests:
     env:
       CHANNEL_GROUP: candidate
       CHURN: "false"
-      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      COMPUTE_MACHINE_TYPE: m5.xlarge
```

Comment:
lets keep the same size here as before and focus only on changing the name of the workflow/chain

Reply:
The instance type isn't correct, Sharada have ever discussed with me before, we need to keep consistency with other rosa upgrade jobs